tests/params/default_emscoring.yaml

```diff
@@ -0,0 +1,297 @@
+#
```

Comment:
Are these the same as the `defaults.yaml`? If so, there is no need to repeat the text here. You can import the defaults directly:

```python
from haddock.modules.scoring.emscoring import DEFAULT_CONFIG as emscoring_default_params
from haddock.gear.yaml2cfg import read_from_yaml_config
```